package.json

```diff
@@ -35,17 +35,23 @@
     "eslint": "^8.16.0",
     "eslint-plugin-astro": "^0.8.1",
     "fast-glob": "^3.2.11",
+    "hastscript": "^7.0.2",
     "htmlparser2": "^7.2.0",
     "kleur": "^4.1.4",
+    "mdast": "^3.0.0",
```

Comment:
Based on the deprecation message, this package has been renamed to `remark`. Can you find a way to make it work without the deprecated package and use remark instead?

Reply:
I’m only using the types. Looked into it and `remark` uses `mdast`’s typings still… really odd. I can get around this by replacing `mdast` with `@types/mdast`. Odd, but works!